x-pack/plugins/security/public/plugin.tsx

```diff
@@ -119,26 +141,13 @@ export class SecurityPlugin
     if (management) {
       this.managementService.start({ management });
     }
-
-    return {
-      __legacyCompat: {
-        account_management: {
-          AccountManagementPage: () => (
-            <core.i18n.Context>
-              <AccountManagementPage
-                authc={this.authc}
-                notifications={core.notifications}
-                apiClient={new UserAPIClient(core.http)}
-              />
-            </core.i18n.Context>
-          ),
-        },
-      },
-    };
   }
 
   public stop() {
-    this.sessionTimeout.stop();
+    if (this.sessionTimeout) {
+      this.sessionTimeout.stop();
+    }
+
     this.navControlService.stop();
```

Comment:
Ah, I see it's because you added a test that will create the plugin and stop it without ever having called setup. So this prevents that test from failing.

Is that a valid execution scenario though? I thought plugins always had to be setup before they could be stopped.

Reply:
> Is that a valid execution scenario though? I thought plugins always had to be setup before they could be stopped.

Long ago when we were discussing NP plugin lifecycle we were assuming that `stop` can safely be called at any point in time, even before plugin is `setup`. It may be very outdated requirement that we don't need to follow now - let's confirm with @elastic/kibana-platform

Reply:
It looks like `stop` can't be called before `setup` ([reference](https://github.com/elastic/kibana/blob/master/src/core/public/plugins/plugin.ts#L136-L139)).

So, perhaps this change should be removed, and the test case should be updated to call `setup` first?

Reply:
> It looks like stop can't be called before setup (reference).

:grimacing: oookay, I agree, will remove this change.